src/app/lib/planship.ts

```diff
@@ -1,4 +1,4 @@
-import { Planship } from '@planship/fetch'
+import { Planship, CustomerSubscriptionWithPlan, ResponseError } from '@planship/fetch'
```

Comment:
It doesn't look like `CustomerSubscriptionWithPlan` and `ResponseError` are used.

Reply:
Fixed.